components/notebook-controller/pkg/metrics/metrics.go

```diff
@@ -0,0 +1,83 @@
+package metrics
+
+import (
+	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+	appsv1 "k8s.io/api/apps/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+// Metrics includes metrics used in notebook controller
+type Metrics struct {
+	cli                  client.Client
+	runningNotebooks     *prometheus.GaugeVec
+	NotebookCreation     *prometheus.CounterVec
```

Comment:
should we provide a method, and not expose the metrics directly?

Reply:
Not necessary I believe

Reply:
I guess we can refactor later if needed

Reply:
I think maybe we will add more metrics for nb-controller. Maybe how many notebooks are culled or some other metrics. So it is not that easy to wrap these into functions. WDYT?